pyproject.toml

```diff
@@ -0,0 +1,80 @@
+[build-system]
+requires = ['hatchling', 'hatch-vcs']
+build-backend = 'hatchling.build'
+
+[project]
+name = 'fmf'
+authors = [
+    { name = 'Petr Splichal', email = 'psplicha@redhat.com' },
+]
+maintainers = [
+    { name = 'Petr Splichal', email = 'psplicha@redhat.com' },
+]
+description = 'Flexible Metadata Format'
+readme = 'README.rst'
+license = 'GPL-2.0-or-later'
+license-files = { paths = ['LICENSE'] }
+requires-python = '>=3.9'
+classifiers = [
+    'Natural Language :: English',
+    'Programming Language :: Python :: 3.9',
+    'Programming Language :: Python :: 3.10',
+    'Programming Language :: Python :: 3.11',
+    'Programming Language :: Python :: 3.12',
+    'Topic :: Utilities',
+]
+keywords = [
+    'metadata',
+    'testing',
+]
+dependencies = [
+    'ruamel.yaml',
+    'filelock',
+    'jsonschema',
+]
+dynamic = ['version']
+
+[project.urls]
+Homepage = 'https://github.com/psss/fmf'
+
+[project.optional-dependencies]
+tests = [
+    'pytest',
+]
+tests-cov = [
+    'fmf[tests]',
+    'pytest-cov',
+]
+dev = [
+    'fmf[tests]',
+    'pre-commit',
+]
+docs = [
+    'sphinx',
+    'sphinx_rtd_theme',
+]
+all = [
+    'fmf[dev]',
+    'fmf[docs]',
+]
+
+[project.scripts]
+fmf = 'fmf.cli:cli_entry'
+
+[tool.hatch]
+version.source = 'vcs'
+version.raw-options.version_scheme = 'post-release'
+
+[tool.hatch.build.targets.wheel]
```

Comment:
I think there should be a `[tool.hatch.build.targets.sdist]` with a list of all files that to pack to be in source tarball.  
Not sure what's currently packaged in fmf, but I would presume things like .docs, .packit.yaml, .readthedocs, .pre-commit-config, etc should not be included.

Reply:
By default `hatch` should be including all git files including hidden files. Could you double-check that? And I think there is no reason not to include all of those files, including stuff like `.pre-commit-config.yaml`. The size difference is minuscule and there are no licensing issues that I am aware of right now.

Reply:
It does include everything, yeah. OK then, I don't know what the packaging guidelines say about this, just pointing there will be more files packaged than there currently are.

Reply:
I think that is intentional, such as a `_version.py` file. Packaging guidelines recommend to not be conservative about it. E.g. the sources from ansible-galaxy are discouraged because they don't contain test files or other upstream files.